### ISSUE TITLE
type:Discussion | Codepen links

### ISSUE THREAD

**Issue by Aniruddha-Shriwant** [reactjs/reactjs.org#4227]
This might not be an issue but I was going through React docs and found this strange behavior

Steps to Reproduce this behavior: 
1. Go to https://reactjs.org/docs/rendering-elements.html#react-only-updates-whats-necessary
2. Click on the "last example" from the second paragraph 

> You can verify by inspecting the last example with the browser tools:

3. That link (https://reactjs.org/redirect-to-codepen/rendering-elements/update-rendered-element) redirects us to 
this link: https://codepen.io/pen?layout=left

So basically the link redirects us to a new codepen tab, where the user has to create a new pen and he/she has to add all the code

Is it supposed to work like this? Or I was thinking of the scenario where after clicking on CodePen links it should redirect to CodePen that has the initial code example discussed in that specific part of documentation.

I'm happy to contribute if this might be an issue.

**Comment by Aniruddha-Shriwant** [comment]
#### Edit : 
I also found that the "Try it on CodePen" link also doesn't work for this page

Click on this link: https://reactjs.org/docs/rendering-elements.html#rendering-an-element-into-the-dom
Click on "Try it on CodePen" it redirects us to the new CodePen

It should redirect us to something similar to this: https://codepen.io/gaearon/pen/GjPyQr?editors=0011 
As this link has pre-written code, I was expecting something similar to this one

**Comment by harish-sethuraman** [comment]
This is codepen links are in other files .
https://github.com/reactjs/reactjs.org/blob/9ed29b0df612b66d4802dba8085b5ef9d52722c8/content/docs/composition-vs-inheritance.md?plain=1#L48 

It seems different in the one you have mentioned.
https://github.com/reactjs/reactjs.org/blob/9ed29b0df612b66d4802dba8085b5ef9d52722c8/content/docs/rendering-elements.md?plain=1#L41

Feel free to raise a PR :)

**Comment by Aniruddha-Shriwant** [comment]
Sure, I will make a PR for this soon :)

**Comment by amenk** [comment]
Same problem in the very first hello world example: https://reactjs.org/docs/hello-world.html

**Comment by Aniruddha-Shriwant** [comment]
> Same problem in the very first hello world example: https://reactjs.org/docs/hello-world.html

@amenk, Thanks for pointing this out, I recently changed to a new distro of Linux, will setup git and my local environment setup within an hour and raise a PR with updated CodePen Links
